### PR TITLE
Update Timeouts.cs

### PR DESCRIPTION
**Pull request by tsvetaevAlex**
update private to readonly

**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

*Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request.*

Please provide enough information so that others can review your pull request:

Explain the **details** for making this change. What existing problem does the pull request solve?

**Test plan (required)**

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

**Code formatting**

**Closing issues**

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: JDI.Light/JDI.Light/Settings/Timeouts.cs [hunk 1]
```diff
@@ -2,9 +2,9 @@
 {
     public class Timeouts
     {
-        private int RetryMSec { get; }
-        private int WaitElementMSec { get; }
-        private int WaitPageLoadMSec { get; }
+        public readonly int RetryMSec;
+        public readonly int WaitElementMSec;
```
Comment (codacy-review) [inline JDI.Light/JDI.Light/Settings/Timeouts.cs:6]: Codacy found an issue: [Make 'WaitElementMSec' private.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=6087138)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=6087138&bid=19927765)

File: JDI.Light/JDI.Light/Settings/Timeouts.cs [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
+        public readonly int WaitPageLoadMSec;
```
Comment (codacy-review) [inline JDI.Light/JDI.Light/Settings/Timeouts.cs:7]: Codacy found an issue: [Make 'WaitPageLoadMSec' private.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=6087138)

File: JDI.Light/JDI.Light/Settings/Timeouts.cs [hunk 3]
```diff
@@ -2,9 +2,9 @@
 {
     public class Timeouts
     {
-        private int RetryMSec { get; }
-        private int WaitElementMSec { get; }
-        private int WaitPageLoadMSec { get; }
+        public readonly int RetryMSec;
```
Comment (codacy-review) [inline JDI.Light/JDI.Light/Settings/Timeouts.cs:5]: Codacy found an issue: [Make 'RetryMSec' private.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=6087138)

**Comment by tsvetaevAlex** [comment]
3 potencial issues found by Codacy

**Pull request closed without merging**